iotilecore/iotile/core/utilities/async_tools/event_loop.py

```diff
@@ -262,6 +263,37 @@ def start(self):
             self.thread = threading.Thread(target=self._loop_thread_main, name="EventLoopThread", daemon=True)
             self.thread.start()
 
+    def wait_for_interrupt(self, check_interval=1.0, max_time=None):
+        """Run the event loop until we receive a ctrl-c interrupt or max_time passes.
+
+        This method will wake up every .1 seconds by default to check for
```

Comment:
Seems to be some discrepancy in the default by the comment (0.1 seconds) and the default arg (1 second)?

Reply:
nice catch, fixed.